subprojects/gradle/instrumentation-tests/src/integTest/kotlin/com/avito/instrumentation/executing/TestExecutorIntegrationTest.kt

```diff
@@ -4,10 +4,10 @@ import com.avito.android.stats.StatsDConfig
 import com.avito.instrumentation.configuration.target.TargetConfiguration
 import com.avito.instrumentation.configuration.target.scheduling.quota.QuotaConfiguration
 import com.avito.instrumentation.createStubInstance
+import com.avito.instrumentation.internal.reservation.devices.provider.KubernetesDevicesProvider
```

Comment:
I vote for an alternative convention:
`com.avito.instrumentation.reservation.devices.provider.internal.KubernetesDevicesProvider`

Created a discussion for this question: https://github.com/avito-tech/avito-android/discussions/736

Reply:
Answered